### PR TITLE
Improves support for window resizing events in IE 11

### PR DESCRIPTION
**Pull request by tonytlwu**
Fixes as suggested by @hcarneiro

### CODE REVIEW

**Comment by tonytlwu** [comment]
@hcarneiro I don't quite approve of this because one scenario fires jQuery event and one fires a native event.

- Does this mean users need to use `window.addEventListener()` or `$.fn.on()` depending on the browser? Or they need to use both? Events should be fired so that users can choose to attach event listeners using one method (and it could be either).
- Is it right to test specifically for MSIE or to run a feature detection script instead?

**Comment by tonytlwu** [comment]
e.g. Based on [this SO article](https://stackoverflow.com/questions/30571488/windows-resize-trigger-event-is-not-working-in-ie11) we could:

```js
if (navigator.userAgent.indexOf('MSIE') !== -1 || navigator.appVersion.indexOf('Trident/') > 0) {
  var evt = document.createEvent('UIEvents');
  evt.initUIEvent('resize', true, false, window, 0);
  window.dispatchEvent(evt);
} else {
  window.dispatchEvent(new Event('resize'));
}
$(window).trigger('resize'); // Or maybe this wouldn't be necessary if firing a native event automatically triggers a jQuery event
```

**Comment by tonytlwu** [comment]
...and here's one that [doesn't appear to rely on User Agent sniffing](https://stackoverflow.com/questions/8712036/dom-event-fired-by-window-resize)

```js
if (document.createEvent) { // W3C
  var ev = document.createEvent('Event');
  ev.initEvent('resize', true, true);
  window.dispatchEvent(ev);
} else { // IE
  element = document.documentElement;
  var event = document.createEventObject();
  element.fireEvent("onresize",event);
}
$(window).trigger('resize'); // Or maybe this wouldn't be necessary if firing a native event automatically triggers a jQuery event
```

**Comment by hcarneiro** [comment]
@tonytlwu This is where I got the solution.
https://stackoverflow.com/questions/30571488/windows-resize-trigger-event-is-not-working-in-ie11

I guess you already answered question **1**
With regards to **2**, I think we need to test for IE 11 and Edge (just in case) or some solution that doesn't require any user-agent...

**Comment by tonytlwu** [comment]
screen shot 2018-06-11 at 17 47 43

@hcarneiro But you've provided a snippet of the code that's in the OP, i.e. doesn't work? Have you tested this? I feel that I'm being suggested a solution that hasn't been fully tested.

**Comment by hcarneiro** [comment]
@tonytlwu I provided this as solution
```
if (navigator.userAgent.indexOf('MSIE') !== -1 || navigator.appVersion.indexOf('Trident/') > 0) {
     var evt = document.createEvent('UIEvents');
     evt.initUIEvent('resize', true, false, window, 0);
     window.dispatchEvent(evt);
    } else {
       window.dispatchEvent(new Event('resize'));

    }
```

And it does work, I did test on a client's app.

**Comment by tonytlwu** [comment]
@hcarneiro What's this then?

screen shot 2018-06-11 at 17 51 18

File: js/build.js [hunk 1]
```diff
@@ -9,6 +9,16 @@
   // ============================
 
   var collapsibleTemplate = Fliplet.Widget.Templates['templates.collapsible'];
+
+  function resizeWindow() {
+    if (Modernizr.windows) {
```
Comment (tonytlwu) [inline js/build.js:14]: @hcarneiro [This solution](https://github.com/Fliplet/fliplet-widget-accordion-start/pull/4#issuecomment-396308614) looks like it directly detects JavaScript features instead of checking for browsers/OS. Have you tried it?
